std/file.d

```diff
@@ -1779,6 +1781,50 @@ if (isInputRange!R && !isInfinite!R && isSomeChar!(ElementEncodingType!R))
     assert(target.timeLastModified(SysTime.min) >= source.timeLastModified);
 }
 
+version(StdDdoc)
+{
+    /++
+     $(BLUE This function is Posix-Only.)
+
+     Returns the time that the given file was last modified.
+     Params:
+     statbuf = statbuf retrieved from file.
```

Comment:
Why `statbuf retrieved from file`? Is `statbuf` a pre-established term? Maybe "`stat_t` retrieved from file"?